### PR TITLE
Add license citation pypi

### PR DESCRIPTION
**Pull request by raar1**
Add a license, CITATION.cff, update README to reflect new repository name, and add github action to auto publish new releases to PyPI.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,9 +25,9 @@ def get_version(rel_path):
 setup(name='fairworkflows',
       version=get_version('fairworkflows/_version.py'),
       description='FAIRWorkflows python library',
-      author='Robin Richardson, Djura Smits',
+      author='Robin Richardson, Djura Smits, Sven van den Burg',
       author_email='r.richardson@esciencecenter.nl',
-      url='https://example.org',
+      url='https://github.com/fair-workflows/FAIRWorkbench/',
```
Comment (svenvanderburg) [inline setup.py:30]: This is incorrect now right? It should be fairworkflows

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+keywords:
+  - "semantic"
+  - "RDF"
+  - "FAIR"
+  - "workflows"
+  - "nanopublications"
+license: "Apache-2.0"
+message: "If you use this software, please cite it using these metadata."
+title: "fairworkflows: A python library for handling semantically described scientific workflows"
+version: "0.1"
```
Comment (svenvanderburg) [inline CITATION.cff:30]: Elsewhere the version is: "0.1.0"

**Review by svenvanderburg — APPROVED** [review]
Seems very sensible to me!

It would be awesome if the tag you tag a release with is also automatically the python version (i.e. "0.1.1"), but I am not sure whether that is easy to setup. This is actually fine!

**Comment by svenvanderburg** [comment]
Maybe wait with merging until we are sure that we want the package to be called `fairworkflows`

File: CITATION.cff (hunk 2 above)
Comment (raar1) [reply CITATION.cff]: Ah yes, good point. I've changed this now.

File: setup.py (hunk 1 above)
Comment (raar1) [reply setup.py]: Indeed, thanks for spotting!

**Pull request merged**